ava.base.js

```diff
@@ -30,6 +30,10 @@ const config = {
   verbose: true,
   timeout: '240s',
   workerThreads: false,
+  extensions: {
```

Comment:
I would do it in each ava.config.js and try to get rid of the base configuration

Reply:
see: https://github.com/netlify/build/pull/4598

Reply:
it does not need to run ts -w as you should import from the sources (as you are now running the tests with typescript)